server/src/test/java/org/elasticsearch/ingest/CompoundProcessorTests.java

```diff
@@ -279,6 +282,41 @@ public void testBreakOnFailure() throws Exception {
         assertStats(pipeline, 1, 1, 0);
     }
 
+    public void testFailurePipelineField() {
```

Comment:
nitpick would it make sense to rename this to what it tests? eg testFailureProcessorIsInvokedOnFailure